addons/website_sale/data/data.xml

```diff
@@ -64,7 +64,7 @@
             <field name="model_id">product.product</field>
             <field name="user_id" eval="False" />
             <field name="domain">[('website_published', '=', True)]</field>
-            <field name="context">{'display_default_code': False}</field>
+            <field name="context">{'display_default_code': False, 'action_rerender': False}</field>
```

Comment:
it is implicit.
since .get() will return False.
But explicit can be better than implicit so ok for me...

Reply:
Not really needed but just thought it could be intresting to leave it there so that if someone copy paste the filter, they could see that we can have an action_rerender in the context.